### DISCUSSION TITLE
How to idiomatically change link hover color?

### DISCUSSION THREAD

**Discussion by AgainstLightning** [saadeghi/daisyui#2062] [Q&A]
Hello I do hope this is a simple question. I am having trouble getting links to change on hover. This is my code, based on what I can find I expect the hover color to be that as defined as "secondary" but it does not work. I am seeing other styles from daisy as well as my custom theme colors showing up, it's this hover I can't get to work. What am I doing wrong?

```
module.exports = {
  content: ["./src/**/*.{js,ts,jsx,tsx,mdx}", "*"],
  important: true,
  daisyui: {
    styled: true,
    themes: [{
      custom: {
        ...require("daisyui/src/theming/themes")["[data-theme=light]"],
        "primary": "#CFF1B2",
        "primary-content": "#333333",
        "secondary": "#EE8F54",
        "accent": "#333333",
        "neutral": "#F5F1EB",
        "base-100": "#FFFFFF",
        "info": " #CBD255",
        "success": "#50AB64",
        "warning": "#FFC353",
        "error": "#D82C0D"
      }
    }],
    base: true,
    utils: true,
    logs: true,
    rtl: false,
    prefix: "",
  },
  presets: [require("@arceo/tailwind-config/tailwind.config.js")]
};
```

```
const Link = () => {
  return (
    <a className={`link hover:secondary`}>I'm a simple link</a>
  )
}

export default Link;
```

**Comment by saadeghi** [comment]
`secondary` is not a class name so it won't do anything.    
It's a color name like Tailwind's `red-500`. Color name itself is not a class name but you can use it like `text-COLORNAME` or `bg-COLORNAME`, etc.  

For secondary color on hover use `hover:text-secondary`
